Uplift/Views/GymDetail/Facilities/MiscellaneousInfoView.swift

```diff
@@ -39,11 +42,11 @@ class MiscellaneousInfoView: UIView {
 
     // MARK: - Helper
 
-    func formatMiscellaneous(_ misc: [String]) -> String {
+    static func formatMiscellaneous(_ misc: [String]) -> String {
         return misc.joined(separator: "\n")
     }
```

Comment:
why are we making this static?

Reply:
`getHeight` needs to be static since it will be called by other classes independent of a `MiscellaneousInfoView` object. `getHeight` also called `formatMiscellaneous` and `generateAttributedString`, so they need to be static too.